app/routers/event.py

```diff
@@ -321,6 +323,25 @@ def add_new_event(values: dict, db: Session) -> Optional[Event]:
         return None
 
 
+def get_tamplate_to_share_event(event_id: int, user_name: str,
+                                db: Session, request: Request) -> templates:
+    """Give yoe nice tamplate of the event that you can share.
+    Get event id and the user name that want to send the share message,
+    Also needed to give db session and request method (mostly 'get')
+    return TemplateResponse that incloud all the ditails on the event"""
+
+    event = by_id(db, event_id)
+    html_temp = templates.TemplateResponse("event/share_event.html",
+                                           {"request": request,
```

Comment:
Can you please format it a bit better? It take lots of lines for no good reason :)